modules/education/src/main/java/edu/stanford/spezi/modules/education/videos/component/ExpandableSection.kt

```diff
@@ -0,0 +1,196 @@
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowUp
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.ElevatedCard
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import edu.stanford.spezi.core.design.theme.Colors
+import edu.stanford.spezi.core.design.theme.Elevations
+import edu.stanford.spezi.core.design.theme.Shapes
+import edu.stanford.spezi.core.design.theme.Spacings
+import edu.stanford.spezi.core.design.theme.SpeziTheme
+import edu.stanford.spezi.core.design.theme.TextStyles
+import edu.stanford.spezi.core.design.theme.TextStyles.titleLarge
+import edu.stanford.spezi.modules.education.videos.Video
+import edu.stanford.spezi.modules.education.videos.VideoItem
+
+@Composable
+fun SectionHeader(
+    text: String?,
+    isExpanded: Boolean,
+    onHeaderClicked: () -> Unit,
+) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .clickable { onHeaderClicked() }
+            .padding(Spacings.medium),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        text?.let {
+            Text(
+                text = it,
+                style = titleLarge
+            )
+        }
+
+        Icon(
+            imageVector = getExpandIcon(isExpanded),
+            contentDescription = if (isExpanded) "Collapse" else "Expand"
+        )
+    }
+}
+
+@Composable
+fun ExpandableSection(
+    modifier: Modifier = Modifier,
+    title: String?,
+    description: String?,
+    videos: List<Video> = emptyList(),
+    expandedStartValue: Boolean = false,
+    onExpand: () -> Unit = {},
+    onActionClick: (Video) -> Unit = { _ -> },
+) {
+    var expanded by remember { mutableStateOf(expandedStartValue) }
+
+    ElevatedCard(
+        elevation = CardDefaults.cardElevation(defaultElevation = Elevations.Card.medium),
+        shape = RoundedCornerShape(Shapes.RoundedCorners.large),
+        colors = CardDefaults.cardColors(
+            containerColor = lightenColor(Colors.surface),
```

Comment:
Can't we just use `Colors.surface.copy(alpha = 0.9f)` instead?

Reply:
alpha 0.9 
![image](https://github.com/StanfordSpezi/SpeziKt/assets/48420258/5d318e87-9d31-4b43-9858-61ec527b0a25)

vs
lighten colors
![image](https://github.com/StanfordSpezi/SpeziKt/assets/48420258/67d456da-8cb0-4820-aef3-6997c74b622a)

vs apha 0.2
![image](https://github.com/StanfordSpezi/SpeziKt/assets/48420258/2a6d560a-3108-44cb-9c06-647c70f97f06)



i had tried that inital, but the problem is that the alpha value only changes the transparency of the colour, but does not actually make the colour brighter, so in my personal opinion it always looked less good. with a few colours the difference is not so big, but if you don't adjust the colours as well, you always have to set the alpha value much lower. with alpha 0.1, for example, you sometimes achieve similar optical results as with 0.9 if you also change the colours. 

for example:
alpha 0.7
![image](https://github.com/StanfordSpezi/SpeziKt/assets/48420258/fce8cf83-eece-490c-8227-5db53bfe061d)

vs lighten 0.7
![image](https://github.com/StanfordSpezi/SpeziKt/assets/48420258/bdc749bc-b79c-4ffd-928f-45dd772b2d97)

Reply:
vs alpha 0.2 
![image](https://github.com/StanfordSpezi/SpeziKt/assets/48420258/a0160926-35da-42be-9480-734f4b58203d)